transifex/pull.py

```diff
@@ -103,8 +104,8 @@ def branch(self):
         """Create and check out a new branch."""
         subprocess.run(['git', 'checkout', '-b', self.branch_name], check=True)
 
-    def pull(self):
-        """Download translated strings from Transifex.
+    def update_translations(self):
```

Comment:
Readability and clarity. This is no longer just pulling updated messages from Transifex.